src/engine/controls/cuecontrol.cpp

```diff
@@ -570,10 +568,7 @@ void CueControl::hotcueSet(HotcueControl* pControl, double v) {
     hotcueClear(pControl, v);
 
     CuePointer pCue(m_pLoadedTrack->createAndAddCue());
-    double closestBeat = m_pClosestBeat->get();
-    double cuePosition =
-            (m_pQuantizeEnabled->toBool() && closestBeat != -1) ?
-                    closestBeat : getSampleOfTrack().current;
+    double cuePosition = getQuantizedCurrentPosition();
```

Comment:
now the cue point is always quantized,  regardless of m_pQuantizeEnabled->toBool()

Reply:
Actually, that part of logic is also moved to `getQuantizedCurrentPosition()`. I guess that is not an adequate name for this function.

Reply:
Really? I don't see that.

Reply:
Right here: https://github.com/mixxxdj/mixxx/blob/1ea29b35789953852c25a8550ab38705fb297165/src/engine/controls/cuecontrol.cpp#L1578

Reply:
Ups, that part was always  there and hidden by GitHub. Sorry. 
The name is OK, no need to change it again.

Reply:
No problem.
Ah OK then.